bagOfWords.py

```diff
@@ -0,0 +1,61 @@
+import re
+import csv
+from bs4 import BeautifulSoup
+from nltk.tokenize import word_tokenize
+from nltk.corpus import stopwords
+from sklearn.feature_extraction.text import CountVectorizer
+
+def removeNonLetter(sentence):
+	filteredSentence = []
+	sentence = word_tokenize(sentence)
+	for word in sentence:
+		if word.isalpha():
+			filteredSentence.append(word)
```

Comment:
```suggestion
    for word in sentence:
        if word.isalpha():
            filteredSentence.append(word)
```
Be careful with your indentations :slightly_smiling_face: Again, nothing serious, but it's part of our conventions

Reply:
Yes!😅